tests/components/wmspro/test_init.py

```diff
@@ -0,0 +1,41 @@
+"""Test the wmspro initialization."""
+
+from unittest.mock import AsyncMock
+
+import aiohttp
+
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+
+async def test_config_entry_device_config(
```

Comment:
Not sure what this is testing

Reply:
The config entry setup, but of course this is then tested again as part of the cover tests. I started with these first, so I guess we could drop the successful one and only keep the failure ones in `test_init.py`